backend/aopp_test.go

```diff
@@ -122,6 +124,7 @@ func TestAOPPSuccess(t *testing.T) {
 			Address:      expectedAddress,
 			CallbackHost: callbackHost,
 			coinCode:     coinpkg.CodeBTC,
+			format:       "any",
```

Comment:
Would be very nice to have one test with a specific format and another one with uknown format to get `errAOPPUnsupportedFormat`.